sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsub/PubsubCoderProviderRegistrar.java

```diff
@@ -31,6 +31,9 @@
   @Override
   public List<CoderProvider> getCoderProviders() {
     return ImmutableList.of(
+        CoderProviders.forCoder(
+            TypeDescriptor.of(PubsubMessage.class),
```

Comment:
I think this change unfortunately will break update feature for runners that support it (for example Dataflow).
cc: @reuvenlax